cs-nutanix-prism/src/test/java/io/cloudslang/content/nutanix/prism/services/AddNicImpTest.java

```diff
@@ -0,0 +1,52 @@
+package io.cloudslang.content.nutanix.prism.services;
+
+import io.cloudslang.content.nutanix.prism.entities.NutanixAddNicInputs;
+import io.cloudslang.content.nutanix.prism.entities.NutanixCommonInputs;
+import io.cloudslang.content.nutanix.prism.service.VMImpl;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static io.cloudslang.content.nutanix.prism.service.VMImpl.createNICURL;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(VMImpl.class)
+public class AddNicImpTest {
+    private static final String EXPECTED_ADD_NIC_PATH = "https://myhost:9080/api/nutanix/v2.0/vms/0b5d5c1c-40c8-4591-9f02-72e2ce/nics";
+
+    private final NutanixAddNicInputs nutanixAddNicInputs = NutanixAddNicInputs.builder()
```

Comment:
Already NICImplTest class is there so shift this code to that class.

Reply:
NICImplTest in this class delete nic code is present so I kept my code in my class